tests-functional/clients/status_backend.py

```diff
@@ -214,6 +215,22 @@ def extract_data(self, path: str):
 
         return self.container.extract_data(path)
 
+    def import_data(self, src_path: str, dest_path: str):
+        """
+        Import a file from the host (src_path) into the container at dest_path.
+        If not running in a container, just copy the file locally.
+        """
+        if not self.container:
+            # Not running in a container, just copy the file locally
+            if os.path.exists(src_path):
+                os.makedirs(os.path.dirname(dest_path), exist_ok=True)
+                with open(src_path, "rb") as src, open(dest_path, "wb") as dst:
+                    dst.write(src.read())
+                return
+            return
+
+        self.container.import_data(src_path, dest_path)
```

Comment:
Looks like will be cleaner inverted:

```suggestion
        if self.container:
            self.container.import_data(src_path, dest_path)

                # Not running in a container, just copy the file locally
        if not os.path.exists(src_path):
            return

        os.makedirs(os.path.dirname(dest_path), exist_ok=True)
        with open(src_path, "rb") as src, open(dest_path, "wb") as dst:
            dst.write(src.read())
        return
```